[management] Network traffic  events docs (#291)
netbirdio/docs: src/pages/how-to/traffic-events-logging.mdx

# Traffic Events Logging

<Note>
 This is an experimental feature. Functionality and behavior may evolve, including changes to how data is collected or reported.
 To use this feature, ensure you have NetBird client version 0.39 or higher.
</Note>

> The traffic events Logging functionality enables comprehensive monitoring and analysis of connections across your infrastructure.

Reviewer: ```suggestion
The traffic events logging functionality enables comprehensive monitoring and analysis of connections across your infrastructure.
```

> It captures various network activities, including peer-to-peer, site-to-site, peer-to-resource, and other critical events.

Reviewer: ```suggestion
It captures network activity, including peer-to-peer, site-to-site, peer-to-resource, and other network traffic events.
```

> It provides detailed visibility into network traffic, helping to answer key questions such as who initiated

Reviewer: ```suggestion
It provides detailed visibility into connections and network traffic flow, helping to answer key questions such as who initiated
```
